Swiftfin/Views/SettingsView/UserDashboardView/DevicesView/DevicesView.swift

```diff
@@ -8,9 +8,12 @@
 
 import Defaults
 import JellyfinAPI
+import OrderedCollections
 import SwiftUI
 
 // TODO: Replace with CustomName when Available
+// TODO: Why doesn't this filter on the user from the Init?
```

Comment:
The getUsers() call from the SDK is filtering or the filtering I added manually? That is, was this resulting in the filtered data Devices on only the selected user:

```
    init(userID: String? = nil) {
        _viewModel = StateObject(wrappedValue: DevicesViewModel(userID))
    }
```

I can give it another go, but I was getting all devices regardless of my userID

Reply:
Ah my bad. Looking at the server code, it looks like the intended use of the endpoint is not to filter on the user id, but to return all devices on the server that the user _could_ connect to, regardless if they ever have. It looks like the server doesn't do what we want it to do, so I may be less inclined to link to devices from users this way.

- https://github.com/jellyfin/jellyfin/blob/a0c634a6edcebdf0ee1ed28fc51dfefaab8b9392/Jellyfin.Server.Implementations/Devices/DeviceManager.cs#L193
- https://github.com/jellyfin/jellyfin/blob/a0c634a6edcebdf0ee1ed28fc51dfefaab8b9392/Jellyfin.Server.Implementations/Devices/DeviceManager.cs#L233

I think overall the device/user system on the server doesn't do what we want it to do since the device doesn't store all past users that have connected through it, but only the last one.

Reply:
That makes way more sense! I mean, we can still link through with 'lastUsername'. Primarily, what I see as valuable is the ability to nuke out all devices for a specific user. I wanted to add that as a toggle for a password reset function as well since this would help with a compromised password. What I have now gets us _pretty close_ to that functionality but it's definitely not 1 to 1 like I thought it was...

Does this still sound valuable or should I remove the User/Devices link?

Reply:
Let's just remove the User -> Devices link. When a user password is reset all old tokens are deleted, so we wouldn't need to nuke the devices to get that functionality.